docs/pages/versions/unversioned/sdk/apple-authentication.md

```diff
@@ -0,0 +1,169 @@
+3. You need to choose a primary app for the Apple Sign In configuration. This is the app whose icon will show up in the Apple Sign In system UI. If you have a set of related apps you might choose the "main" app as the primary, but most likely you'll want to just use the app you're working on now as the primary.
+4. In the list of identifiers, click on the one corresponding to your primary app. Enable the "Sign In with Apple" capability, click "Edit", and choose the "Enable as a primary App ID" option. Save the new configuration.
+5. If you chose a different app to be the primary, you'll also need to open up the configuration page for your current app, enable the "Sign In with Apple" capability, click "Edit" and choose the "Group with an existing primary App ID" option. Save this configuration as well.
+6. Finally, go to the "Keys" page and register a new key. Add the "Sign In with Apple" capability, and make sure to choose the correct primary app on the configuration screen.
+
+## API
+
+```ts
+import * as AppleAuthentication from 'expo-apple-authentication';
+```
+
+### Methods
+
+- [`AppleAuthentication.isAvailableAsync()`](#appleauthenticationisavailableasync)
+- [`AppleAuthentication.signInAsync(options)`](#appleauthenticationsigninasyncoptions)
+- [`AppleAuthentication.getCredentialStateAsync(user)`](#appleauthenticationgetcredentialstateasyncuser)
+
+### Components
+
+- [`AppleAuthentication.AppleAuthenticationButton`](#appleauthenticationappleauthenticationbutton)
+
+### Prop Types
+
+- [`AppleAuthentication.AppleAuthenticationButtonProps`](#appleauthenticationappleauthenticationbuttonprops)
+
+### Enum Types
+
+- [`AppleAuthentication.AppleAuthenticationButtonStyle`](#appleauthenticationappleauthenticationbuttonstyle)
+- [`AppleAuthentication.AppleAuthenticationButtonType`](#appleauthenticationappleauthenticationbuttontype)
+- [`AppleAuthentication.AppleAuthenticationCredentialState`](#appleauthenticationappleauthenticationcredentialstate)
+- [`AppleAuthentication.AppleAuthenticationScope`](#appleauthenticationappleauthenticationscope)
+- [`AppleAuthentication.AppleAuthenticationStatus`](#appleauthenticationappleauthenticationstatus)
+- [`AppleAuthentication.AppleAuthenticationUserDetectionStatus`](#appleauthenticationappleauthenticationuserdetectionstatus)
+
+### Object Types
+
+- [`AppleAuthentication.AppleAuthenticationCredential`](#appleauthenticationappleauthenticationcredential)
+- [`AppleAuthentication.AppleAuthenticationFullName`](#appleauthenticationappleauthenticationfullname)
+- [`AppleAuthentication.AppleAuthenticationSignInOptions`](#appleauthenticationappleauthenticationsigninoptions)
+
+### Errors
+
+- [Error Codes](#error-codes)
+
+## Methods
+
+### `AppleAuthentication.isAvailableAsync()`
+
+Determine if the current device's operating system supports Apple authentication.
+
+#### Returns
+
+A Promise that resolves to `true` if the system supports Apple authentication, and `false` otherwise.
```

Comment:
```suggestion
A promise that resolves to `true` if the system supports Apple authentication, and `false` otherwise.
```

(Prefer lowercasing common types like this, similar to how we wouldn't talk about returning an "Object" with a capital O.)